jupyter_notebook/README.md

```diff
@@ -0,0 +1,81 @@
+# Jupyter Notebook
+
+This directory contains the Jupyter notebook support for
+the FlexFlow Python binding.
+
+## Quick Start
+### Pre-requisite
+* Python >= 3.6
+* FlexFlow Python binding needs to be installed, please check the [installtion guide](https://github.com/flexflow/FlexFlow/blob/master/INSTALL.md)
+* Install Jupyter notebook
+
+        pip install notebook
+
+### Install the FlexFlow IPython kernel
+```
+python ./install.py --(configurations)
+```
+Please refer to the [Kernel Configurations](#kernel-configurations) section for the configuration details.
+
+If the installtion is successed, the following log will be printed to the terminal.
+The `flexflow_kernel_nocr` is the kernel name, and the `FlexFlow_SM_GPU` is the display name
+of the kernel, which can be modified by the configuration json file. 
+`FlexFlow` is the name entry in the json file, `SM` means the kernel
+is only for shared memory machine, and `GPU` meams GPU execution is enabled. 
+```
+IPython kernel: flexflow_kernel_nocr(FlexFlow_SM_GPU) has been installed
+```
+The installed kernel can be also seen by using the following command:
+```
+jupyter kernelspec list
+```
+
+### Start the Jupyter Notebook server
```

Comment:
It would probably be best to do this in two steps: first explain how you'd do everything just on the compute node, and then explain the setting you'd want to use a separate compute node and then detail how to do it